specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/EnterpriseKnowledgeGraphSwagger.json

```diff
@@ -0,0 +1,2001 @@
+            }
+          },
+          "interpretationResult": {
+            "$ref": "#/definitions/InterpretationResult"
+          },
+          "restatement": {
+            "type": "string"
+          },
+          "suggestions": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/EntityDocumentSuggestionChip"
+            }
+          }
+        }
+      },
+      "ErrorResponse": {
+        "type": "object",
+        "properties": {
+          "error": {
+            "$ref": "#/definitions/ErrorMessage"
+          }
+        }
+      },
+      "ErrorMessage": {
+        "type": "object",
+        "properties": {
+          "code": {
+            "type": "string"
+          },
+          "message": {
+            "type": "string"
+          }
+        }
+      },
+      "EntityDocument": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string"           
+          },
+          "types": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/EntityType"
+            }   
+          },
+          "dominantType": {
+            "$ref": "#/definitions/EntityType"
+          },
+          "properties": {
+            "type": "object",
+            "additionalProperties": {
+              "uniqueItems": false,
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/EntityNode"
+              }
+            }   
+          },
+          "timeStamp": {
+            "format": "date-time",
+            "type": "string"
+          },
+          "sourceIds": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": {
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        }
+      },
+      "InterpretationResult": {
+        "type": "object",
+        "properties": {
+          "utterance": {
+            "$ref": "#/definitions/Utterance"
+          },
+          "sparql": {
+            "$ref": "#/definitions/SparqlQuery"
+          },
+          "confidence": {
+            "format": "double",
+            "type": "number"
+          },
+          "mainProjection": {
+            "type": "string"
+          },
+          "mainProjectionEntityType": {
+            "type": "string"
+          },
+          "restatement": {
+            "type": "string"
+          }
+        }
+      },
+      "EntityDocumentSuggestionChip": {
+        "type": "object",
+        "properties": {
+          "actionEntityDocument": {
+            "$ref": "#/definitions/EntityDocument"
+          },
+          "suggestionInterpretation": {
+            "$ref": "#/definitions/InterpretationResult"
+          },
+          "actionEntityId": {
+            "type": "string"
+          }
+        }
+      },
+      "EntityType": {
```

Comment:
Why do you have a model item with only a single property? Is the intent for the number of attributes to grow over time? If so, should attributes named `entityType` be of type "/definitions/EntityType` or should the attribute name be `entityTypeName`?